build.gradle.kts

```diff
@@ -143,7 +146,8 @@ subprojects {
          project.name.endsWith("-benchmarks") ||
          project.name.contains("golden") ||
          project.name.contains("protoc") ||
-         project.displayName.contains("sample")
+         project.displayName.contains("sample") ||
+         project.displayName.contains("wire-tests-swift")
```

Comment:
we do not want to apply checkStyle under this folder for anything, this was causing Gradle configuration to fail.

Reply:
That's on me to followup here